compose/install.md

```diff
@@ -117,6 +117,78 @@ on Microsoft Windows Server and want to install Docker Compose.
 
 ### Install Compose on Linux systems
 
+You can install Docker Compose in different ways, depending on your needs:
+
+- In testing and development environments, some users choose to use automated
+  [convenience scripts](#install-using-the-convenience-script) to install Docker.
+
+- Most users
+  [set up Docker's repositories](#install-using-the-repository) and install
+  from them, for ease of installation and upgrade tasks. This is the
+  recommended approach.
+
+- Some users download the binary and
+  [install it manually](#install-binary-manually) and manage
+  upgrades completely manually.
```

Comment:
I think the blank lines between bullets should not be there (otherwise each bullet is formatted as a paragraph and additional whitespace)

With blank lines;
<img width="968" alt="Screenshot 2022-05-02 at 13 20 23" src="https://user-images.githubusercontent.com/1804568/166225955-9d634cfb-c732-49f7-a8b2-277523be5f8a.png">


Without;

<img width="980" alt="Screenshot 2022-05-02 at 13 09 28" src="https://user-images.githubusercontent.com/1804568/166225948-d7b49f63-52e4-43a7-bb1c-a42861bdb774.png">

Perhaps change the "Some users download the binary and install it manually and manage upgrades completely manually." to "download and install the binary", and manage ...


```suggestion
- In testing and development environments, some users choose to use automated
  [convenience scripts](#install-using-the-convenience-script) to install Docker.
- Most users [set up Docker's repositories](#install-using-the-repository) and
  install from them, for ease of installation and upgrade tasks. This is the
  recommended approach.
- Some users [download and install the binary](#install-binary-manually),
  and manage upgrades manually.
```

(if we think there should be more space in between by default, we should adjust the theme/stylesheet 😅 )